docs/api.md

```diff
@@ -4124,3 +4252,30 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 ## Success (object)
 + success: `true` (boolean, required)
 
+## Dataset (object)
++ id: `1` (number, required) - The numerical ID of the Dataset.
++ name: `people` (string, required) - The name of the Dataset
++ createdAt: `2018-01-19T23:58:03.395Z` (string, required) - ISO date format.
++ projectId: `1` (number, required) - The numerical ID of the Project that the Dataset belongs to.
++ revisionNumber: `1` (number, required) - The revision number of the Dataset. It is incremented whenever the Dataset's properties are modified.
```

Comment:
Is there an updatedAt?

Reply:
not yet. 

I have removed `revisionNumber` and `id` from the documentation as we are not returning them in API anymore